telegram/bot.py

```diff
@@ -2051,3 +2426,12 @@ def __reduce__(self):
     sendInvoice = send_invoice
     answerShippingQuery = answer_shipping_query
     answerPreCheckoutQuery = answer_pre_checkout_query
+    restrictChatMember = restrict_chat_member
+    promoteChatMemmber = promote_chat_member
```

Comment:
Mistype, should be `promoteChatMember` (that's why `tests.test_official` [fails](https://travis-ci.org/python-telegram-bot/python-telegram-bot/jobs/248911924#L1403) here)